source/includes/_organizations.html.md

```diff
@@ -176,6 +176,25 @@ admin_only | false | If set to true, the result will only include administrator
 
 [Paginated](#collection-pagination)
 
+## Remove member
+
+```shell
+curl \
+  -H "Accept: application/vnd.api+json"
+  -H "Authorization: Token token={TOKEN}"
+  -X DELETE \
+  https://api.codeclimate.com/v1/orgs/596b70adb79d8f147b000002/members/516342ca7e11a428b0025372
+```
+Removes the given user from the account members. Be aware that this endpoint is enabled on the demand. You should conctact our customer support team in order to be able to consume this endpoint.
```

Comment:
"is enabled on the demand" >> 
"is enabled on demand"

Reply:
add "(help@codeclimate.com)" after the last sentence (telling them to contact us).